test_data/rules/join_return_with_assignment.dart

```diff
@@ -24,15 +24,16 @@ int doubleFoo() {
 }
 
 class A {
-  int _a;
+  late int _a;
   int get myA {
+    // ignore: dead_null_aware_expression
```

Comment:
Could we get rid of this one by making `_a` nullable?

Reply:
Yep. Done.